examples/platforms/nrf52840/openthread-core-nrf52840-config.h

```diff
@@ -137,6 +137,6 @@
   * Define to 1 if you want use legacy transmit done.
   *
   */
-#define OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE 1
+#define OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE 0
```

Comment:
Please remove this config option from nRF52840 platform, since it is not a regular config option, the legacy transmit done callback related codes will be totally removed after all platforms could pass ACK frame.